src/filters/inspect.c

```diff
@@ -2157,6 +2157,169 @@ static void dump_temi_time(GF_InspectCtx *ctx, PidCtx *pctx, FILE *dump, const c
 	}
 }
 
+static void scte35_parse_splice_time(GF_InspectCtx *ctx, FILE *dump, GF_BitStream *bs)
+{
+	Bool time_specified_flag = gf_bs_read_int(bs, 1);
+	if (time_specified_flag == 1) {
+		/*reserved = */gf_bs_read_int(bs, 6);
+		u64 pts_time = gf_bs_read_int(bs, 33);
+		DUMP_ATT_LLU("pts_time", pts_time);
+	} else
+		/*reserved = */gf_bs_read_int(bs, 7);
+}
+
+static void dump_scte35_info_m2ts_section(GF_InspectCtx *ctx, PidCtx *pctx, FILE *dump, const char *pname, const GF_PropertyValue *att)
+{
+	if (ctx->xml) {
+		inspect_printf(dump, " <SCTE35");
+	} else {
+		inspect_printf(dump, " SCTE35");
+	}
+
+	if (!pctx->bs)
+		pctx->bs = gf_bs_new(att->value.data.ptr, att->value.data.size, GF_BITSTREAM_READ);
+	else
+		gf_bs_reassign_buffer(pctx->bs, att->value.data.ptr, att->value.data.size);
+
+	uint8_t table_id = gf_bs_read_u8(pctx->bs);
```

Comment:
`uint8_t` currently doesn't build on windows

maybe we should add an `#include <stdint.h>` in the windows part of `setup.h`?

or maybe we should just use the `u8` type instead?

Reply:
I will use u8, thanks! FYI this is just a draft PR ATM. I'm not even sure we will not cherry-pick the commits on here.